octopusdeploy_framework/resource_azure_container_registry_test.go

```diff
@@ -18,49 +17,50 @@ type azureFeedTestData struct {
 	password     string
 }
 
+// TODO: uncomment the below test cases when deploy v2025.2 is released for download.
```

Comment:
See https://app.shortcut.com/octopusdeploy/story/108632/uncomment-tests-when-deploy-v2025-2-is-released-for-self-hosted-customers.